tests/post/steps/conftest.py

```diff
@@ -6,6 +6,44 @@
     when,
     parsers,
 )
+import yaml
+
+
+# Pytest fixtures
+@pytest.fixture(scope="module")
+def kubeconfig_data(request, host):
+    """Fixture to generate a kubeconfig file for remote usage."""
+    with host.sudo():
+        kubeconfig_file = host.file('/etc/kubernetes/admin.conf')
```

Comment:
dunno where you can specify an encoding, but if possible specify one (since relying on the default one may bite us in the future).

Reply:
Tracked [here](https://github.com/scality/metalk8s/issues/781#issuecomment-476565028).